wls-gui-wahllokalsystem/src/util/rules.ts

```diff
@@ -38,3 +38,18 @@ export const TIME_GREATER_OR_EQUAL =
 export const TIME_LESS_OR_EQUAL = (compareValue: string) => (value: string) =>
   getDateFromTimeString(value) <= getDateFromTimeString(compareValue) ||
   `Eingabe muss kleiner oder gleich ${compareValue} sein.`;
+
+export const DATE_NOT_BEFORE_WAHLTAG = (
+  value: string,
+  currentUserWahltag: string
+) => {
+  const valueDate = parseDateString(value);
+  const wahltagDate = parseDateString(currentUserWahltag);
+
+  if (valueDate === null || wahltagDate === null) {
+    return "Ungültiges Datumsformat.";
+  }
+  return (
+    valueDate >= wahltagDate || `Eingabe darf nicht vor dem Wahltag liegen.`
+  );
+};
```

Comment:
würde ich generischer formulieren und nicht konkret auf den wahltag beziehen damit man auch andere vergleiche machen kann

```suggestion
export const DATE_NOT_BEFORE = (
  value: string,
  date: string
) => {
  const valueDate = parseDateString(value);
  const compareDate= parseDateString(date);

  if (valueDate === null || compareDate === null) {
    return "Ungültiges Datumsformat.";
  }
  return (
    valueDate >= compareDate || `Eingabe darf nicht vor dem ${compareDate.toLocaleDateString()} liegen.`
  );
};
```